src/components/navbar.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable arrow-body-style */
```

Comment:
- [ ] Let's not disable eslint rules. These are here to help you write best code ever.